prov/efa/src/rxr/rxr.h

```diff
@@ -226,6 +226,7 @@ enum rxr_lower_ep_type {
 enum rxr_x_entry_type {
 	RXR_TX_ENTRY = 1,
 	RXR_RX_ENTRY,
+	RXR_PKT_ENTRY,
```

Comment:
this name is not very descriptive.  Let's try better, with some documentation.

Reply:
I new revision, I created a new enum ```rxr_read_context_type``` and added more comments about each type context.